GrandCentralBoard/Widgets/Bonus/View/BonusWidgetViewModel.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  Created by Krzysztof Werys on 24/02/16.
+//  Copyright © 2016 Krzysztof Werys. All rights reserved.
+//
+
+import Foundation
+
+struct BonusWidgetViewModel {
+    let sceneModel: BonusSceneModel
```

Comment:
ViewModel is not a mere capsule for the model. Please extract data to render and expose as properties. You can initialize ViewModel with model (but please keep it as extension).